### PR TITLE
Add segment

### PR DESCRIPTION
**Pull request by tjclavel**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Cleanup
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Test Plan

- [ ] Unit test
- [ ] Tested locally

### CODE REVIEW

File: src/layers/comms/index.tsx [hunk 1]
```diff
@@ -60,7 +63,7 @@ const Comms = ({ children }: Children) => {
     } else {
       const missedHandler = registry.find(h => h.origin === origin)
       if (missedHandler) {
-        // TODO: Log unknown iframe message from known source
+        track('Unrecognized message', { origin, message: data })
```
Comment (tjclavel) [inline src/layers/comms/index.tsx:66]: for now, just added one `track` to make sure the infra works

File: src/layers/analytics/utils.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const createAnalytics = (environment: 'development' | 'production') => {
```
Comment (tjclavel) [inline src/layers/analytics/utils.ts:1]: mostly copy-pasta except for switching on `env`

File: src/layers/analytics/index.tsx [hunk 3]
```diff
@@ -11,17 +12,35 @@ type AnalyticsLayerContext = {
...
+  useEffect(() => {
+    !window.parafinAnalytics && createAnalytics(environment)
+  }, [environment])
+
+  const identify = useCallback((userId: string) => {
+    try {
+      window.parafinAnalytics!.identify(userId)
+    } catch (err) {
+      // TODO: Throw analytics error
+      console.log('analytics error')
```
Comment (reedhodder) [inline src/layers/analytics/index.tsx:27]: console.log here, not sure if you wanted to keep it

**Review by reedhodder — APPROVED** [review]

File: src/layers/analytics/index.tsx (hunk 3 above)
Comment (tjclavel) [reply src/layers/analytics/index.tsx:27]: ah, thanks

**Pull request merged**